### DISCUSSION TITLE
Dragging a polygon's vertex

### DISCUSSION THREAD

**Discussion by DmitryEvstakhov** [Mapsui/Mapsui#1901] [Q&A] [answered]
Hi guys!
I use example Mapsui.Examples.Mpf.Editing. When I dragging point from a polygon I have some problems:
1. Dragging vertex is not smooth. I mean that the vertex from the VertexOnlyLayer drags smoothly, but the rendered geometry of polygon is not smooth (jerky).
2.  Sometimes the geometry doesn't redraw at all. You have moved one vertex of polygon, but the polygon rendered geometry remains the same. But when I move the map the geometry will be rendered correctly.

Initial geometry:
![01](https://user-images.githubusercontent.com/59865880/227527846-9e796f0c-30d2-4e6a-b112-93ef6761f61d.png)

When I dragged vertex:
![02](https://user-images.githubusercontent.com/59865880/227528006-1d097422-de3c-4872-a56e-c493a19ab00e.png)

When I move the map:
![03](https://user-images.githubusercontent.com/59865880/227528324-9f4a9265-4411-46d2-b238-ef1c21b97e12.png)

Why is this happening and how to solve this problem?

**Comment by DmitryEvstakhov** [comment]
video is more informative
https://user-images.githubusercontent.com/59865880/227552763-b25cfccf-2b4a-477d-b6be-eb210a2b56f4.mp4

**Comment by pauldendulk** [comment, marked as answer]
@DmitryEvstakhov Thanks for reporting this. I can reproduce the problem and turned it into an issue https://github.com/Mapsui/Mapsui/issues/1902. It is a problem related to caching.
